changelog/J/JesusChristTeachingsof.md

```diff
@@ -0,0 +1,24 @@
+# What Has Changed In the “JesusChristTeachingsof” Article
+
+## Major changes 
+The following sentence was difficult to simplify:
+> _Jewish apocalyptic literature (…)_
+
+It was therefore extrapolated as such:
+> _Jewish apocalyptic literature (i.e., a prophetic-like style – …)_
```

Comment:
Perhaps `prophecy-like style of writing` would be better. Don't you think?

Reply:
I think "a prophetic style" covers it because "style" already shows that we're describing "something like" prophecy rather than some narrow definition.

Reply:
Pushed

Reply:
Oups wait! Not in the changlog

Reply:
Done